helm/prefect-server/README.md

```diff
@@ -57,3 +63,23 @@ This endpoint must be accessible by *the user* of the UI, not just the server, s
 If you are using the subchart deployed database with persistence enabled, 
 it is likely the password for the user has persisted between deployments in the PVC for the database but the secret has been regenerated by the Helm chart and does not match. 
 Deploy and set the 'postgresql.existingSecret' option or set a constant password at `postgresql.postgresqlPassword`.
+
+### The agent is running but when I look at the logs I am getting GraphQL errors
+
+The following error is typically caused by the lack of a tenant. Create a default tenant and the agent will restart automatically.
+```
+[2020-11-13 19:16:02,083] ERROR - agent | 400 Client Error: Bad Request for url: http://my-release-apollo.default:4200/graphql
+
+This is likely caused by a poorly formatted GraphQL query or mutation. GraphQL sent:
+
+query {
+    mutation($input: get_runs_in_queue_input!) {
+            get_runs_in_queue(input: $input) {
+                flow_run_ids
+        }
+    }
+}
+variables {
+    {"input": {"before": "2020-11-13T19:16:01.973734+00:00", "labels": ["n", "u", "l", "l"], "tenant_id": null}}
```

Comment:
Does `null` / `None` really get parsed like this into individual characters?  If so, that seems like a bug?  The default should be `"labels": []`

Reply:
I agree the default should be changed. This must be the intended behavior of toJson 🤷 weird though.

Is this what we want?
```
       - name: PREFECT__CLOUD__AGENT__LABELS
          value: "[\"foo\",\"bar\"]"
```

Reply:
(good catch btw)

Reply:
Classic javscript; yup `"[\"foo\",\"bar\"]"` looks good and `"[]"` as default is correct 👍